gitspy/Repository.swift

```diff
@@ -17,4 +17,55 @@ struct Repository {
     let newStars: Int
     let totalStars: Int
     
+    // MARK: Network
+
+    static func repos(userId: String, completion: (parsedRepos: [Repository]?) -> Void) {
+        let userResource = Resource(pathComponent: "repos?user_id=" + userId) { (object) -> [Repository]? in
+            guard let dictionary = object as? [String: AnyObject],
+                let repos = dictionary["repos"] as? [[String: AnyObject]] else {
+                return nil
+            }
+            
+            return Repository.array(repos)
+        }
+        
+        Networker.sharedInstance.loadResource(userResource, callback: completion)
+    }
+    
+    static func watch(repoId: Int, completion: (status: Bool?) -> Void) {
+        let boolResource = Resource(pathComponent: "repos/" + String(repoId), method: "POST") { (object) -> Bool? in
+            guard let status = object["status"] as? String else {
+                return nil
+            }
+
+            return status == "ok"
+        }
+        
+        Networker.sharedInstance.loadResource(boolResource, callback: completion)
+    }
+
+    init(language: String, name: String, description: String, id: Int, newStars: Int, totalStars: Int) {
+        self.language = language
+        self.name = name
+        self.description = description
+        self.id = id
+        self.newStars = newStars
+        self.totalStars = totalStars
+    }
```

Comment:
I added it for completeness, to call it from other initializers. Since you lose this default initializer when you create another one